ci-operator/config/opendatahub-io/opendatahub-operator/opendatahub-io-opendatahub-operator-feature-operator-refactor.yaml

```diff
@@ -7,7 +7,7 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.21
+    tag: golang-1.22
```

Comment:
if the original PR is only targeting incubation, we can drop this one. soon feature-oprator-refactor will be killed